### DISCUSSION TITLE
What happen when use a method not available in the old version in a plugin interceptor

### DISCUSSION THREAD

**Discussion by Crearns** [apache/skywalking#7473] [Q&A] [answered]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?
- If a plugin use a method or a class not available in the old version, and I use this plugin intercept the old version，it will throw Error ？
- 如果插件引用的依赖是新版本，并且在拦截的时候使用一个老版本没有的方法，那么当老版本的方法被拦截的时候会抛出类或方法找不到的错误吗？

**Comment by wu-sheng** [comment, marked as answer]
Definitely, which is caused by JVM level check as always. Why do you ask this?
